### ISSUE TITLE
Add ability to define footer row(s)

### ISSUE THREAD

**Issue by mr-m-1t** [oleksandrbalan/lazytable#17]
Hello :wave: 
This library looks pretty slick, I'd love to use it in my project.

A common use case for tables is to display a footer, for example to show totals or averages:

<details>
  <summary>Example - click to expand</summary>

| student | quiz 1 | quiz 2 | final exam |
| ------- | ------ | ------ | ---------- |
| Alice   | 78%    | 84%    | 85%        |
| Bob     | 87%    | 87%    | 85%        |
| Carol   | 92%    | 87%    | 78%        |
| Dave    | 88%    | 83%    | 82%        |
| ------- | ------ | ------ | ---------- |
| average | 86%    | 85%    | 82%        |
</details>

Is it feasible to extend this project to allow a fixed footer row, similar to how a fixed header row is can be set?
Perhaps something like this:
<details>
  <summary>Example - click to expand</summary>

```kt
public fun lazyTablePinConfiguration(
    columns: Int = 0,
    rows: Int = 0,
): LazyTablePinConfiguration { ... }

// could become
public fun lazyTablePinConfiguration(
    columns: Int = 0,
    rows: Int = 0,
    // the count of pinned footer rows
    endRows: Int = 0,
): LazyTablePinConfiguration { ... }
```
</details>

I did see #7, and the snippet provided there could be adapted to achieve this behavior. However showing a footer row is a very common use case for tables, and as such I believe it makes sense to add the capability to this library.

**Comment by oleksandrbalan** [comment]
Hi, thx 🤗

Yea, I agree it would be nice feature for `LazyTable`, I will mark your issue as an enchantment and will look on it when I have time. Unfortunately I cannot promise any timings :confused:

**Comment by mr-m-1t** [comment]
Awesome!
My team may be able to allocate some time for this, would you be open to a PR adding footer functionality?

**Comment by oleksandrbalan** [comment]
Sure thing 👍

**Comment by mattgraham1** [comment]
@oleksandrbalan We created a PR (https://github.com/oleksandrbalan/lazytable/pull/19) for these changes. For whatever reason I'm unable to add you as a reviewer so I'm pinging you here hoping you see it. Please review it when you have time and let us know if you see any needed changes. Thank you!

**Comment by oleksandrbalan** [comment]
Awesome, thank you very much! I will try to look on it as soon as possible 👀

**Comment by mattgraham1** [comment]
Following up to see if you'll be able to review this PR. I'd really appreciate it and please let us know if there are any concerns. Thank you very much!

**Comment by oleksandrbalan** [comment]
@mattgraham1 Sorry for the late response, I have checked it and everything works fine 👍 
Even the iOS configuration (for me at least 😅). 

<img width="519" alt="image" src="https://github.com/oleksandrbalan/lazytable/assets/20944869/69ae7b7f-5b31-4a69-95e0-15278fbb2c5b">

There are just few nitpicks in PR from me, thx!

**Comment by mattgraham1** [comment]
Thanks you for checking iOS. I'm 98% Android developer so probably a setup issue with iOS on my side. I've address all the PR feedback you gave. Thanks again!

**Comment by oleksandrbalan** [comment]
Available in [1.7.0](https://github.com/oleksandrbalan/lazytable/releases/tag/v1.7.0) :clap:  :rocket:
